raaghu-components/src/rds-comp-typing-section/rds-comp-typing-section.tsx

```
@@ -95,141 +97,130 @@ const RdsCompTypingSection = (props: RdsTypingAltProps) => {
-                                displayType="Icon + Text"
-                                icon="figma_icon"
-                                label="Import From Figma"
-                                textCase="unset"
-                                tooltipPlacement={
-                                    TooltipStyle.LeftArrow
-                                }
-                                tooltipTitle="This is tooltip"
-                            />
-                        </div>
+        <div className ="border-radius transperent">
+        <div className="d-flex flex-column typing-section px-1 pt-1">
```

ESLint

[error] Expected indentation of 12 spaces but found 8.

(indent)